sos/policies/distros/redflag.py

```diff
@@ -0,0 +1,50 @@
+# This Python file uses the following encoding: utf-8
+
+# 版权所有（c）Yang Qing <yangqing@redflag-os.com>
+#
+# 此文件是sos项目的一部分：https://github.com/sosreport/sos
+#
+# 该受版权保护的材料可供任何希望使用的人使用，
+# 根据以下条款和条件对其进行修改、复制或重新分发
+# GNU通用公共许可证的第2版。
+#
+# 有关详细信息，请参阅源发行版中的LICENSE文件。
```

Comment:
Can you update this based on the guidelines, and can you put this in English please